src/ScheduleService.Models.Dto/Requests/Group/DeleteGroupRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+
+public class DeleteGroupRequest
```

Comment:
you don't need class for single property. Just pass Guid into command

Reply:
+